builder/comp-builder.nix

```diff
@@ -173,7 +173,7 @@ let
       "--with-ghc-pkg=${ghc.targetPrefix}ghc-pkg"
       "--with-hsc2hs=${ghc.targetPrefix}hsc2hs"
     ] ++ lib.optional (pkgConfigHasPrefix && pkgconfig != [])
-      "--with-pkg-config=${ghc.targetPrefix}pkg-config"
+      "--with-pkg-config=${buildPackages.cabalPkgConfigWrapper.targetPrefix}pkg-config"
```

Comment:
Hamish can you explain the logic for this? What's this pkgConfigHasPrefix thing? where does that come from?

Reply:
I have removed `pkgConfigHasPrefix` it was only needed for nixpkgs <20.09pre